oneanddone/tests/functional/conftest.py

```diff
@@ -71,3 +76,10 @@ def task(base_url, is_local):
     if is_local:
         from oneanddone.tasks.tests import TaskFactory
         return TaskFactory.create()
+
+
+@pytest.fixture(scope='session')
```

Comment:
Why have you chosen session scope? This means that multiple tests using this fixture in a single session will use the same task. It would be better for all tests to create their own task.